duo-example-admin/src/main/java/com/duosecurity/example/DuoPolicies.java

```diff
@@ -106,15 +108,15 @@ private static void printGlobalPolicy(CommandLine cmd) {
     }
   }
 
-  private static void createNewPolicy(CommandLine cmd) {
+  private static String createNewPolicy(CommandLine cmd) {
     // Create a new policy and print it to stdout.
```

Comment:
I think we should update this doc string to reflect that this method now also return policy key as string

Reply:
Done, updated all the docstrings in the example to make it consistent.